api/tests/core/finance/test_api.py

```diff
@@ -581,12 +638,32 @@ def test_cancelled_after_use(self):
         motive = models.FinanceEventMotive.BOOKING_CANCELLED_AFTER_USE
         booking = bookings_factories.CancelledBookingFactory()
 
-        event = api.add_event(booking, motive=motive)
+        event = api.add_event(motive=motive, booking=booking)
 
         assert event.booking == booking
         assert event.status == models.FinanceEventStatus.NOT_TO_BE_PRICED
         assert event.motive == motive
 
+    @pytest.mark.parametrize(
+        "incident_motive",
+        [
+            models.FinanceEventMotive.INCIDENT_REVERSAL_OF_ORIGINAL_EVENT,
+            models.FinanceEventMotive.INCIDENT_NEW_PRICE,
+            models.FinanceEventMotive.INCIDENT_COMMERCIAL_GESTURE,
+        ],
+    )
+    def test_create_incident_event(self, incident_motive):
+        pricing_point = offerers_factories.VenueFactory()
+        booking_incident = factories.IndividualBookingFinanceIncidentFactory(
+            booking__stock__offer__venue__pricing_point=pricing_point, incident__venue__pricing_point=pricing_point
+        )
+
+        event = api.add_event(incident_motive, booking_incident=booking_incident)
+
+        assert event.bookingFinanceIncident
```

Comment:
[détail] On pourrait même écrire : 

```suggestion
        assert event.bookingFinanceIncident == booking_incident
```